src/rbindlist.c

```diff
@@ -82,7 +82,7 @@ SEXP rbindlist(SEXP l, SEXP usenamesArg, SEXP fillArg, SEXP idcolArg, SEXP ignor
       if (!length(cn)) continue;
       const SEXP *cnp = STRING_PTR_RO(cn);
       for (int j=0; j<thisncol; j++) {
-        SEXP s = cnp[j];
+        SEXP s = ENC2UTF8(cnp[j]); // convert different encodings for use.names #5452
```

Comment:
I've lost my familiarity with all of `rbindlist()`, why do we need to do `ENC2UTF8()` 3 times?